src/mono/mono/mini/mini-amd64.c

```diff
@@ -3415,9 +3415,9 @@ simd_type_to_shl_op (int t)
 	case MONO_TYPE_I:
 	case MONO_TYPE_U:
 #if TARGET_SIZEOF_VOID_P == 8
-		return OP_PSHLD;
```

Comment:
Is the #if really needed ?

Reply:
Probably not needed. Since it is amd64, `MONO_TYPE_U` and `MONO_TYPE_I` should be 64 bits integer.